src/javascript/_common/language.js

```diff
@@ -45,7 +45,7 @@ const Language = (() => {
 
     const getLanguage = () => {
         if (/ach/i.test(current_lang) || /ach/i.test(languageFromUrl())) {
-            const crowdin_lang_key = 'jipt_language_code_binary-static';
+            const crowdin_lang_key = 'jipt_language_code_deriv-app';
```

Comment:
@negarn Not sure if this would break anything, just let me know if I need to revert it